coherence/doc.go

```diff
@@ -534,6 +535,115 @@ Lastly, when you create a Customer object you must set the Class value matching
 	    log.Fatal(err)
 	}
 
+# Using Near Caches
+
+The Coherence Go client allows you to specify a near cache to cache frequently used and accessed data in your Go application.
+When you access data using Get() or GetAll() operations, returned entries are stored in the near cache and subsequent data
+access for keys in the near-cache is almost instantaneous.
```

Comment:
That a near-cache will make data access near instant whereas without a near-cache, each operation results in a network call?